tests/core/test_model.py

```diff
@@ -562,6 +563,62 @@ def value_max(cls, df: pd.DataFrame) -> Iterable[bool]:
         schema.validate(df, lazy=True)
 
 
+def test_registered_dataframe_checks(
+    extra_registered_checks,
+):  # pylint: disable=unused-argument
+    """Check that custom check inheritance works"""
+    # pylint: disable=unused-variable
+
+    @pax.register_check_method(statistics=["one_arg"])
+    def base_check(df, *, one_arg):
+        # pylint: disable=unused-argument
+        return True
+
+    @pax.register_check_method(statistics=["one_arg", "two_arg"])
+    def child_check(df, *, one_arg, two_arg):
+        # pylint: disable=unused-argument
+        return True
+
+    # pylint: enable=unused-variable
+
+    check_vals = {"one_arg": 150, "two_arg": "hello"}
+
+    class Base(pa.SchemaModel):
+        a: Series[int]
+        b: Series[int]
+
+        class Config:
+            no_param_check = ()
+            base_check = check_vals["one_arg"]
+
```

Comment:
To be extra cautious, we could add a third check whose argument are overwritten by `Child`.

Reply:
Good idea